src/resources/team/team.service.ts

```diff
@@ -167,3 +153,20 @@ export const getItems = async (
 
 	return items
 }
+
+export const deleteItem = async (item: ItemDocument): Promise<ItemDocument> => {
+	logger.debug(`Delete item with id: ${item.id}`)
+
+	return item.remove()
+}
+
+export const updateItem = async (
+	item: ItemDocument,
+	itemUpdate: ItemInput,
+): Promise<ItemDocument> => {
+	logger.debug(`Update item with id: ${item.id}`)
+
+	_.merge(item, itemUpdate)
```

Comment:
I would say this is an insecure way of doing update. We never know what is in request body. So probably better sending only `itemId` and `itemUpdate` to this service and use `findOneAndUpdate` to do the update

Reply:
I've already extracted the fields that we need in the controller, so itemUpdate is save now